packages/gatsby-source-graphql/src/gatsby-node.js

```diff
@@ -0,0 +1,137 @@
+const crypto = require(`crypto`)
+const uuidv4 = require(`uuid/v4`)
+const { buildSchema, printSchema } = require(`graphql`)
+const {
+  makeRemoteExecutableSchema,
+  transformSchema,
+  introspectSchema,
+  RenameTypes,
+} = require(`graphql-tools`)
+const { createHttpLink } = require(`apollo-link-http`)
+const fetch = require(`node-fetch`)
+const invariant = require(`invariant`)
+const {
+  NamespaceUnderFieldTransform,
+  StripNonQueryTransform,
+} = require(`./transforms`)
+
+exports.sourceNodes = async (
+  { boundActionCreators, createNodeId, cache, store },
```

Comment:
I tough `boundActionCreators` where deprecated in flavor of `actions`??

Reply:
You are right, luckily this is just matter of renaming `boundActionCreators` to `actions` :)

Reply:
Done